OpenSimRT/Common/include/CircularBuffer.h

```diff
@@ -45,6 +46,7 @@ template <int history, typename T> class CircularBuffer {
         current = 0;
         startOver = false;
         newValue = false;
+        externalNotification = false;
```

Comment:
This is to notify the consumer thread and remove the `conditional_variable` from the wait state in case where no new data are added to the buffer. It was added to terminate the program normally when the simulation ends. I've added a comment mentioning this.